### PR TITLE
Use PHP 5.5 syntax.

### LINKED ISSUE #8: Php 5.5 support
It seems that this package is supported only by PHP 5.6 and 7. Can you please add support at least for  PHP version 5.5.35

Thank you.

### PR DESCRIPTION
**Pull request by Arantor**
This should fix the only obvious issues in Slim-Bridge that prevent it working on PHP 5.5. Fixes #8 

I have only given it basic testing, it seems to work for my colleagues (I don't use it myself)

### CODE REVIEW

**Comment by mnapoli** [comment]
Thank you, could you update `.travis.yml` so that tests runs on PHP 5.5?

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
         }
     },
     "require": {
-        "php": "~5.6|~7.0",
+        "php": ">=5.5.0",
```
Comment (mnapoli) [inline composer.json]: Could you keep the `~5.5|~7.0` notation? That avoids speculating on BC breaks in PHP 8

**Comment by Arantor** [comment]
I've updated it - though I've never used Travis so I don't know if that's the correct way to do it.

**Comment by mnapoli** [comment]
Travis configuration is correct, tests are failing because on invalid PHP code in `config.php`: https://travis-ci.org/PHP-DI/Slim-Bridge/jobs/141813495#L265

Also it seems that PHPUnit's version isn't compatible with PHP 5.5: https://travis-ci.org/PHP-DI/Slim-Bridge/jobs/141813494#L156 (you might need to change PHPUnit's version)

**Comment by Arantor** [comment]
Ugh. The check it fails is that this all hinges on PHPUnit, which also requires 5.6, and I doubt you'd merge a patch in that would cause the build to always fail as far as Travis is concerned.

Guess this was a waste of time :disappointed: but I'll leave the pull request up in case anyone who wants PHP 5.5 support can somehow make use of it. We're not all in the happy position to be using current PHP versions, unfortunately.

**Comment by mnapoli** [comment]
@Arantor you can change PHPUnit's version (use PHPUnit 4.*) so that it works with PHP 5.5

**Comment by mnapoli** [comment]
Replaced by #14

**Pull request closed without merging**